common/src/main/java/com/wynntils/models/players/type/GuildRank.java

```diff
@@ -5,20 +5,30 @@
 package com.wynntils.models.players.type;
 
 public enum GuildRank {
-    RECRUIT("Recruit"),
-    RECRUITER("Recruiter"),
-    CAPTAIN("Captain"),
-    STRATEGIST("Strategist"),
-    CHIEF("Chief"),
-    OWNER("Owner");
+    RECRUIT("Recruit", ""),
+    RECRUITER("Recruiter", "★"),
+    CAPTAIN("Captain", "★★"),
+    STRATEGIST("Strategist", "★★★"),
+    CHIEF("Chief", "★★★★"),
+    OWNER("Owner", "★★★★★");
 
     private final String name;
+    private final String stars;
 
-    GuildRank(String name) {
+    GuildRank(String name, String stars) {
         this.name = name;
+        this.stars = stars;
     }
 
     public String getName() {
         return name;
     }
+
+    public String getGuildDescription() {
+        if (this == RECRUIT) {
+            return name();
+        } else {
+            return name() + " (" + stars + ")";
```

Comment:
Should this just be `name`? With `name()` it returns the rank name in full caps, I think it looks cleaner with just the first letter being capitalised

Reply:
Yeah, you are right! My mistake. `name()` is the intrinsic enum method. It should be `getName()` or `name`; chose whichever variant you prefer.